check-licenses/action.yml

```diff
@@ -0,0 +1,34 @@
+name: "Check licenses"
+description: "Check if package licenses in the current environment are among the accepted ones"
+
+inputs:
+  accepted-licenses:
+    description: "List of accepted licenses"
+    default: "MIT BSD Apache PSF HPND"
+    required: false
+    type: string
+
+runs:
+  using: "composite"
+  steps:
+
+    - name: "Check licences of packages"
+      shell: bash
+      run: |
+        echo ${{ inputs.accepted-licenses }}
+        pip install pip-licenses
+        input=${{ inputs.accepted-licenses }}
+        accepted_licenses=''
+        for license in ${input[*]}; do accepted_licenses+="$license\|"; done
+        echo $accepted_licenses
+        accepted_licenses=${accepted_licenses::-2}
+        echo $accepted_licenses
+        echo pip-licenses
```

Comment:
No need to echo a CLI tool that already outputs to stdout.

Reply:
Thats why I added both, with echo (in line 26) and without it (in line 27), just to see if any of them was printing in the logs

Reply:
I've removed the unnecessary echos. However, it looks like it outputs to stdout an empty string:

![image](https://user-images.githubusercontent.com/53979143/203967729-c5b9f97d-fedc-4bc6-b877-318bd8c056f2.png)

I am able to print the ``accepted_licenses`` variable (strings to find in the ``pip_licenses --summary``) before and after removing the last two characters. But then I get 3 empty lines for the 3 prints that was supposed to show.

Reply:
The issue is that you have a clean Python environment without any package installed in the CI.

Reply:
It is working fine. Try to install numpy and matplotlib, so the Python environment gets cluttered with dependencies and packages.

Reply:
If you do so, just add a comment to explain why we need matplotlib and numpy.
@jorgepiloto Could numpy be enough?

Reply:
Maybe I'm wrong, but this was just to test the action. Now I'm going to add all the libraries for PyAnsys2023.1.1 and I will remove the matplotlib and numpy imports. It shouldn't be required anymore when calling the action from other repos.

Reply:
Yes, AFAIK, it was only for testing purposes.